toqito/state_props/is_separable.py

```diff
@@ -148,9 +155,26 @@ def is_separable(state: np.ndarray, dim: None | int | list[int] = None, level: i
     # Check these tests.
     if min_dim == 2:
         # Check if X is separable from spectrum.
-        if (lam[0] - lam[2 * max_dim - 1]) ** 2 <= 4 * lam[2 * max_dim - 2] * lam[2 * max_dim] + tol**2:
-            print("Determined to be separable by inspecting its eigenvalues.")
-            print("N. Johnston. Separability from spectrum for qubit-qudit states. Phys. Rev. A, 88:062330, 2013.")
+        if (lam[0] - lam[2 * max_dim - 2]) ** 2 <= 4 * lam[2 * max_dim - 3] * lam[2 * max_dim - 1] + tol**2:
+            return True
+
+        # For the rest of the block matrix tests, we need the 2-dimensional
+        # subsystem to be the *first* subsystem, so swap accordingly.
+        state_t = swap(state, [1, 2], dim) if dim[0] > 2 else state
+
+        # Check if Lemma 1 of refs{13} applies to X. Also check the Hildebrand 2xn results.
```

Comment:
What's refs{13} ?

Reply:
Thanks, had forgotten to remove that while porting from the MATLAB code.